v3/src/components/graph/adornments/adornment-component-info.ts

```diff
@@ -0,0 +1,27 @@
+import React from "react"
+
+export interface IAdornmentComponentInfo {
+  adornmentEltClass: string
+  Component: React.ComponentType<any>
+  inspectorList: {
+    position: number
+    label: string
+    hint: string
+  }
```

Comment:
Does it make sense to include information like this here for use in building the adornments list in the Inspector panel?

Reply:
I don't think so. I think the inspector panel knows about the adornments, but not vice versa.

Reply:
I'm inclined to say this should be part of the registration system. We may want to  call it something more generic than `inspectorList` -- I might eliminate the hierarchy entirely and just have something like:
```typescript
export interface IAdornmentComponentInfo {
  type: string
  adornmentEltClass: string
  Component: React.ComponentType<any>
  order: number
  label: string
  hint: string
}
```
Otherwise, the inspector code would need to import all of the adornment constants individually, which can be avoided if the required information is registered in this way.